src/job/jobHandler.ts

```diff
@@ -635,27 +655,43 @@ export abstract class JobHandler {
     return this._logger;
   }
 
-  protected async previewWebhook(): Promise<AxiosResponse<string>> {
-    const previewWebhookURL = 'https://webhook.gatsbyjs.com/hooks/data_source';
-    const githubUsername = this.currJob.user;
-    const gatsbySiteId = await this._repoEntitlementsRepo.getGatsbySiteIdByGithubUsername(githubUsername);
-    if (!gatsbySiteId) {
-      const message = `User ${githubUsername} does not have a Gatsby Cloud Site ID.`;
-      this._logger.warn('Gatsby Cloud Preview Webhook', message);
-      throw new Error(message);
-    }
+  // Invokes Gatsby Cloud Preview Webhook
+  protected async callGatsbyCloudWebhook(): Promise<void> {
```

Comment:
Logic should be relatively untouched. Consolidating the logic from 2 separate functions into 1 since the logic no longer needs to live in 2 different places.